### PR TITLE
Added location config in GCS Done File Marker Plugin

### PR DESCRIPTION
**Pull request by itsankit-google**
[PLUGIN-965](https://cdap.atlassian.net/browse/PLUGIN-965)

Before:- 
Screen Shot 2021-10-29 at 10 51 03 PM

After:-
Screen Shot 2021-11-02 at 11 49 41 AM

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSDoneFileMarker.java [hunk 1]
```diff
@@ -119,6 +120,13 @@ public void run(BatchActionContext batchActionContext) throws IOException {
     @Macro
     public String path;
 
+    @Name(NAME_LOCATION)
+    @Macro
+    @Nullable
+    @Description("The location where the GCS bucket will get created. " +
+      "This value is ignored if the bucket already exists.")
+    protected String location;
```
Comment (seanzhougoogle) [inline src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSDoneFileMarker.java]: why protected ?

File: widgets/GCSDoneFileMarker-postaction.json [hunk 2]
```diff
@@ -81,6 +73,27 @@
           "name": "serviceAccountJSON"
         }
       ]
+    },
+    {
+      "label": "Auto Create",
```
Comment (seanzhougoogle) [inline widgets/GCSDoneFileMarker-postaction.json:78]: this group name is wired, did you confirm with UX ?

File: src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSDoneFileMarker.java (hunk 1 above)
Comment (itsankit-google) [reply src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSDoneFileMarker.java]: Changed to private.

**Review by seanzhougoogle — APPROVED** [review]

File: widgets/GCSDoneFileMarker-postaction.json (hunk 2 above)
Comment (itsankit-google) [reply widgets/GCSDoneFileMarker-postaction.json:78]: Discussed offline to keep it the same as followed by other plugins.

**Pull request merged**